script/demoScripts/demoRelay.ts

```diff
@@ -0,0 +1,173 @@
+import deployments from '../../deployments/arbitrum.staging.json'
+import {
+  RelayFacet__factory,
+  ILiFi,
+  type RelayFacet,
+  ERC20__factory,
+} from '../../typechain'
+import { ethers, utils } from 'ethers'
+import dotenv from 'dotenv'
+dotenv.config()
+
+const main = async () => {
+  const RPC_URL = process.env.ETH_NODE_URI_ARBITRUM
+  const PRIVATE_KEY = process.env.PRIVATE_KEY
+  const LIFI_ADDRESS = deployments.LiFiDiamond
+
+  const provider = new ethers.providers.JsonRpcProvider(RPC_URL)
+  const signer = new ethers.Wallet(PRIVATE_KEY as string, provider)
+  const relay = RelayFacet__factory.connect(LIFI_ADDRESS, provider)
+
+  const address = await signer.getAddress()
+
+  let tx
+
+  // Bridge ETH
```

Comment:
Please also add a Swap&Bridge example and the executed transaction link.

Reply:
https://arbiscan.io/tx/0xfb390520a478912f1bb3df39259fc0bca122aad4891ab15195bfe29c2c7a1086